src/types/app.ts

```diff
@@ -84,6 +85,7 @@ export const integrationFieldProperties: {
   transmission: ['username', 'password'],
   jellyfin: ['username', 'password'],
   plex: ['apiKey'],
+  pihole: ['password'],
```

Comment:
Maybe changing this from password to apiKey could help make it a bit clearer.
I accidentally used my admin password during testing lol 😅
```suggestion
  pihole: ['apiKey'],
```

Reply:
No, the required "password" is the web-password - not an API key